openra/templates/search.html

```diff
@@ -1,37 +1,51 @@
+{% include 'header.html' %}
+
 <div class='cBlock'>
-{% if global_search_request.maps.amount == 0 %}
-Nothing found
+{% if search_results.total == 0 %}
+<h3>
+    Nothing found
+</h3>
 {% else %}
-	{% if global_search_request.maps.hash %}
-		<p>Matches in Map HASH:</p>
-		{% for item in global_search_request.maps.hash %}
-			<a href="/maps/{{item.id}}/"><img src="/maps/{{item.id}}/minimap" /></a><br />
+    <div class="search-group">
+	{% if search_results.map_hash %}
+        <h3>Matches in Map HASH:</h3>
+		{% for item in search_results.map_hash %}
+            {% include 'search_map.html' %}
```

Comment:
This doesn't seem right either.

Reply:
(Same for the occurrences below.)

Reply:
Ok these all use tabs now instead of spaces (as that seems to be the convention here)

Reply:
Ok these all use tabs now instead of spaces (as that seems to be the convention here)